### ISSUE TITLE
Should aria-rowindex/colindex really set the position?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1386]
https://www.w3.org/TR/wai-aria-1.2/#aria-rowindex:

> Defines an element's row index or position with respect to the total number of rows within a table, grid, or treegrid. 

Is with position the actual position meant, as it can be defined e.g. also with aria-owns deviating from the order in the source code. Then a line with rowindex=1 should be output by the screen reader before the line with rowindex=2, even if the latter is located before the former in the source code.

Or should only the position be output by the screen reader as a number, but the order of output follows the order in the source code?

If some lines have aria-rowindex and others do not: What implicit index do the lines without aria-rowindex have?

See: https://github.com/FreedomScientific/VFO-standards-support/issues/488

**Comment by JAWS-test** [comment]
The question is relevant because JAWS behaves differently here than NVDA:
* NVDA outputs all rows in their source code order regardless of aria-rowindex.
* JAWS skips certain rows, e.g. with the same aria-rowindex or with aria-rowindex larger than the largest implicit rowindex. I.e. JAWS does not output certain rows

See https://codepen.io/jaws-test/pen/oNYvaRj

**Comment by AutoSponge** [comment]
IMO, it should only replace native information about index/count. Where AT would normally describe "row X of Y" it will instead use these attributes. It should be considered an anti-pattern to use `aria-rowindex` inconsistently or out of order. The primary purpose is to update virtualized/paginated tables with correct index info.

**Comment by jnurthen** [comment]
Proposal:
This needs to be retested.
We should then add some notes advising authors how to use it and AT how to consume it.
